CMakeLists.txt

```
@@ -1154,22 +1157,26 @@ IF (ENABLE_FILTER_ZSTD)
   FIND_PACKAGE(Zstd)
 ENDIF()
 
+
 # Accumulate standard filters
 set(STD_FILTERS "deflate") # Always have deflate*/
 set_std_filter(Szip)
 SET(HAVE_SZ ${Szip_FOUND})
 set_std_filter(Blosc)
 IF(Zstd_FOUND)
 set_std_filter(Zstd)
+
 ENDIF()
 IF(Bz2_FOUND)
   set_std_filter(Bz2)
+ELSEIF(BZIP2_FOUND)
+  set_std_filter(BZIP2)
```

I don't know what set_std_filter does but it doesn't look like something I would change when I touch the dependency lookup. And there shouldn't be `Bz2_FOUND` after the other suggestion. Last not least the result variable has a different spelling.
```suggestion
IF(BZip2_FOUND)
  set_std_filter(Bz2)
```